keyboards/launchpad/keymaps/via/config.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#define RGBLED_NUM 10
+#define RGB_DI_PIN F6
+#define RGBLIGHT_ANIMATIONS
```

Comment:
Is there a reason these are not defined at the keyboard level?

Reply:
The board doesn't have RGB built-in, as it's done w/strips.

Reply:
So, are you restricted to using F6 and 10 LEDs when using VIA then? Or can it be configured?

Reply:
It can't be configured in via, so, as it is now, if you'd like to use it without compiling on cli, you'll have to use f6 and 10 leds. Open to suggestions on how to resolve this.

Reply:
If using RGB strips is a fairly "standard" upgrade for this PCB (in terms of the pin used and the number of LEDs), then it's probably OK to just turn on RGBLight at the keyboard level in this case, then it's available for QMK Configurator and user built firmware by default.